script.wellbeing/addon.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.wellbeing" name="Wellbeing" version="1.0.0" provider-name="Heckie">
+	<requires>
+		<import addon="xbmc.python" version="3.0.0" />
+	</requires>
+	<extension point="xbmc.service" library="addon.py" />
+	<extension point="xbmc.python.script" library="script.py" />
+	<extension point="xbmc.addon.metadata">
+		<summary lang="en_GB">Wellbeing</summary>
```

Comment:
Make my brain happy with a fix in indentation here :)

Reply:
I formatted it again but I can't see any difference. What is wrong with the indentation?